### PR TITLE
feat: per-server quotapoints

### PR DESCRIPTION
**Pull request by nyapat**
for now, with the ts-ignore calls, ~~it works *with* the old data (i think, i had to delete it to make it work)~~ (does not)

### CODE REVIEW

**Comment by dmansdman** [comment]
Fix ESLINT code scan

File: src/commands/logging/AddQuotaPoints.ts [hunk 1]
```diff
@@ -50,16 +50,7 @@ export class AddQuotaPoints extends BaseCommand {
      */
     public async run(ctx: ICommandContext): Promise<number> {
         const mStr = ctx.interaction.options.getString("member", false);
-        const points = ctx.interaction.options.getInteger("points", false) ?? 20;
```
[the remark this answers is not in the document]
Comment (ewang2002) [reply src/commands/logging/AddQuotaPoints.ts:53]: copy and paste go brrr

File: src/commands/general/Stats.ts [hunk 2]
```diff
@@ -149,11 +149,12 @@ export class Stats extends BaseCommand {
 
         const pts = await LoggerManager.getPoints((resMember as IResolvedMember).member);
         let description = `${EmojiConstants.TICKET_EMOJI} Points: ${pts}`;
-        if(stats.quotaPoints > 0) {
-            description +=`\nQuota Points: ${stats.quotaPoints}`;
+        const serverQuotaPoints = stats.quotaPoints.find(x => x.key === ctx.guild?.id);
+        if (serverQuotaPoints) {
+            description += `\nQuota Points: ${stats.quotaPoints}`;
```
Comment (ewang2002) [inline src/commands/general/Stats.ts:154]: This is a certified `[object Object]` moment 

image

**Review by ewang2002 — CHANGES_REQUESTED** [review]
Didn't test it fully, but please see my one comment.

**Comment by ewang2002** [comment]
lmao perfect. Thanks, I'll continue looking.

@nyapat one more thing -- can you bump the version to 1.3.1? Might as well do that in this PR since it's open

**Review by ewang2002 — APPROVED** [review]
Looks good to me

**Pull request merged**